utils/shared.js

```diff
@@ -1492,6 +1492,39 @@ const filterSelectedFields = (dataObjArray, selectedFieldsArray) => {
     });
 }
 
+const getDatetimeForEmailLink = () => {
+    const currentDate = new Date();
+    const year = currentDate.getUTCFullYear();
+    const month = currentDate.toLocaleString("default", {
+        timeZone: "UTC",
+        month: "long",
+    });
+    const day = currentDate.getUTCDate();
+    const hours = currentDate.getUTCHours();
+    const minutes = currentDate.getUTCMinutes();
+
+    return `${year} ${month} ${day < 10 ? "0" + day : day} ${
+        hours < 10 ? "0" + hours : hours
+    }:${minutes < 10 ? "0" + minutes : minutes} Z`;
+};
+
+const getTemplateForEmailLink = (email, continueUrl) => {
+    return `
+    <html>
+    <head></head>
+    <body marginheight="0">
+      <p>Hello,</p>
+      <p>We received a request to sign in to Connect for Cancer Prevention Study using this email address, at ${getDatetimeForEmailLink()}. If you want to sign in with your ${email} account, click this link:</p>
+      <p><a href="${continueUrl}" target="_other" rel="nofollow">Sign in to Connect for Cancer Prevention Study</a></p>
+      <p>If you did not request this link, you can safely ignore this email.</p>
+      <p>Thanks,</p>
+      <p>Your Connect for Cancer Prevention Study team</p>
+    </body>
+    </html>
+  `;
+};
+
+const NIHMailbox = 'NCIConnectStudy@mail.nih.gov'
```

Comment:
This is minor and doesn't involve functionality but code styling. At first glance, I was going to suggest renaming `NIHMailbox` to `nihMailbox` to adhere to camelcase. However, I think it's fine since NIH is an acronym.

Reply:
I agree that we should use `nihMailbox`.